init/common.js

```diff
@@ -145,7 +153,7 @@ const setupGoogleAuthProvider = async () => {
 
 const createInterests = async (userId) => {
   const service = strapi.service("api::interest.interest");
-  let existing = await service.findOne({});
+  let existing = await service.findOne(userId);
```

Comment:
That's not how findOne works, if you want to find existing interest by userId, you will need to use findMany with filters parameter https://docs.strapi.io/dev-docs/api/entity-service/crud#findmany